cache/search_stats.go

```diff
@@ -135,6 +135,8 @@ type SearchStatsCache interface {
 	IsTagWithEnoughUnits(uid string, count int, cts ...string) bool
 	IsSourceWithUnits(uid string, cts ...string) bool
 	IsSourceWithEnoughUnits(uid string, count int, cts ...string) bool
+	IsTagHasSingleUnit(uid string, cts ...string) bool
```

Comment:
How often is this updated

Reply:
no new update in cache data is added

Reply:
Not sure I follow if there is a new lesson.... After what time is the carousel be showen?

Reply:
currently it is set to 1 minute